java/struts2/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.struts.benchmarker</groupId>
+    <artifactId>struts2</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>war</packaging>
+    <name>struts2</name>
+
+    <properties>
+        <struts2.version>2.5.22</struts2.version>
```

Comment:
could you use an interval (I mean `[2.5,2.6)`)  ?

Reply:
I tried but then Maven uses `2.6-SNAPSHOT` version which is a development version. Being specific in Maven based project is a good approach.

Reply:
Ah ... I haven't the same problem with other `java` frameworks, but I have with `clojure` and `kotlin`. There is not way to specify a range **BUT** avoid any `development`, `rc` ... versions in **maven** ?

Reply:
It supposed to ignore `-SNAPSHOT` versions but somehow Maven includes it, I was testing locally and still the same.

```
Resolution of dependency ranges should not resolve to a snapshot (development version) unless it is included as an explicit boundary.
```

Reply:
`mvn clean package`

```
dev-1:[1.8] ~/P/O/w/j/struts2 (master) > ll target/struts2-1.0/WEB-INF/lib
total 10936
-rw-r--r--+ 1 lukaszlenart  staff    72446 Apr 10 07:38 commons-fileupload-1.4.jar
-rw-r--r--+ 1 lukaszlenart  staff   214788 Apr 10 07:38 commons-io-2.6.jar
-rw-r--r--+ 1 lukaszlenart  staff   523372 Apr 19 10:23 commons-lang3-3.10.jar
-rw-r--r--+ 1 lukaszlenart  staff   207030 Apr 10 07:38 commons-text-1.8.jar
-rw-r--r--+ 1 lukaszlenart  staff  1702975 Apr 19 10:23 freemarker-2.3.30.jar
-rw-r--r--+ 1 lukaszlenart  staff   777727 Apr 10 07:39 javassist-3.24.1-GA.jar
-rw-r--r--+ 1 lukaszlenart  staff   292301 May 16 08:19 log4j-api-2.13.3.jar
-rw-r--r--+ 1 lukaszlenart  staff   264012 Apr 19 10:23 ognl-3.2.14.jar
-rw-r--r--+ 1 lukaszlenart  staff  1530698 Jun 22 08:12 struts2-core-2.6-SNAPSHOT.jar
```

Reply:
Just added the version range as it works with 2.6 as well.

Reply:
I was tricked liked that before (either with `maven` or `gradle`)